doc/src/tutorials/arch/heterogeneous_tiles.rst

```diff
@@ -0,0 +1,189 @@
+.. _heterogeneous_tiles_tutorial:
+
+Heterogeneous tiles tutorial
+============================
+
+This tutorial aims at providing information to the user on how to model sub tiles to enable *heterogeneous tiles* in VPR.
+
+An *heterogeneous tile* is a tile that includes two or more site types that may differ in the following aspects:
+
+- *Fc* definition
+- *Pin locations* definition
+- *IO ports* definition
+
+As a result, an *heterogeneous tile* has the possibility of having multiple block types at the same *x* and *y* location in the grid.
+This comes with the introduction of a third spatial coordinate that identifies the placement of the block type within the x and y grid coordinate.
+
+Moreover, the placer can choose and assign different locations for each block type within the same coordinates as well.
+
+.. figure:: sub_tiles_grid.png
+
+    Device grid, with x, y and sub block coordinates. Each block can be moved by the placer in all the three spatial dimensions.
+
+To correctly model an architecture, each :ref:`arch_tiles` requires at least one sub tile definition. This represents a default
+homogeneous architecture, composed of one or many capacity instances of the sub tile within the physical tile.
+
+To enhance the expressivity of VPR architecture, additional sub tiles can be inserted alongside with the default sub tile.
+This enables the definition of the *heterogeneous tiles*.
+
+With this new capability, the device grid of a given architecture does include a new ``depth`` coordinate that identifies
```

Comment:
Should be ``subblock`` instead of depth for consistency?

Reply:
Done